momoIOS/ViewController.swift

```diff
@@ -57,4 +61,12 @@ class ViewController: UIViewController {
     @objc private func goToAdminViewController() {
         self.navigationController?.pushViewController(AdminSessionTableViewController(), animated: true)
     }
+    
+    @objc private func goToNewNexterViewController() {
```

Comment:
```suggestion
    @objc private func goToNewNextersViewController() {
```
NIT: 오타?! (사실 임시 ViewController라 상관 없긴 할듯)

Reply:
맞당 ~~~ 밑에거 수정하면서 수정할게요 😽